index.html

```diff
@@ -671,16 +671,36 @@ <h4>OAuth 2.0</h4>
     <section>
       <h3>Service Instances</h3>
       <p>
-Service configuration is determined by administrators on per instance basis.
-Instance configuration can include, but is not limited to, credential format, key options, credential status mechanisms, and/or credential templates.
-Administrators are expected to configure service instances such that `options` included in client requests cannot result in incorrect action or problematic responses by the service.
+The APIs defined in this specification presume that they are attached to a
+specific instance of some kind with an associated configuration that has been
```

Comment:
To me, the "of some kind" text is superfluous
```suggestion
specific instance with an associated configuration that has been
```